### ISSUE TITLE
N1 won't connect after computer wakes up

### ISSUE THREAD

**Issue by sachinag** [nylas/N1#3015]
Sometimes when my computer resumes from suspend, N1 subsequently fails to reconnect.  I have to restart N1 to get it to re-establish a connection to get my mail.

**Comment by Bergg** [comment]
Yes, sounds familiar - probably a duplicate of a previously mentioned issue.

**Comment by jackiehluo** [comment]
@sachinag Are you seeing the same as https://github.com/nylas/N1/issues/2243?

**Comment by morgana1723** [comment]
I have the same issue. What I see is actually: 
![screen shot 2017-06-28 at 9 18 41 am](https://user-images.githubusercontent.com/29778923/27693397-fe21fc80-5cae-11e7-85f3-67bf5f873199.png)

So frustrating! It started to happen two weeks ago or so for me and I have to reconnect everyday. I've written support and email and contacted through twitter, but I had no replies so far.
